src/utils.js

```diff
@@ -3,6 +3,18 @@ export function extract(obj, path, dflt) {
   return path.split('.').reduce((obj, key) => (obj || $n)[key], obj) || dflt;
 }
 
+export function isEmpty(value) {
+  let length = 0;
```

Comment:
In this case, wouldn't it be better to have multiple early `return`s instead of reassigning values?